roles/make-applier-projects-unique/tasks/param-file-regex-replace.yml

```diff
@@ -4,6 +4,6 @@
     regexp: "{{ pair.regexp }}"
     replace: "{{ pair.replace }}"
   with_items:
-  - "{{ regexp_replace_pairs }} "
+  - "{{ params_regexp_replace_pairs }} "
```

Comment:
let's add | default([]) in this variable to avoid it bumming out if this variable isn't defined

Reply:
hmmm I want to provide the end user a failure message, as these are end user supplied variables. ` | default([])` will cause this to fail silently. perhaps the answer is to provide a `fail`?